lxc/completion.go

```diff
@@ -1549,7 +1549,7 @@ func (g *cmdGlobal) cmpStoragePoolVolumeSnapshots(poolName string, volumeName st
 
 // cmpStoragePoolVolumes provides shell completion for storage pool volumes.
 // It takes a storage pool name and returns a list of storage pool volumes along with a shell completion directive.
-func (g *cmdGlobal) cmpStoragePoolVolumes(poolName string) ([]string, cobra.ShellCompDirective) {
+func (g *cmdGlobal) cmpStoragePoolVolumes(poolName string, customOnly bool) ([]string, cobra.ShellCompDirective) {
```

Comment:
A boolean param that's used as a constant everywhere the function is called is a code smell; can you refactor this into `cmpStoragePoolVolumes` and `cmpStoragePoolCustomVolumes` (or similar)?

E.g. when I run into this in the wild, I have zero context for what `true` means here. There's no connection at the call site between the parameter's value and its meaning.
```go
volumes, compdir := g.cmpStoragePoolVolumes(pool, true)
```

Reply:
I get your point, but this would result in a lot of code duplication, which is something we are always trying to avoid. I can extend the function comment and then it would be clear what 'true' means just by looking at the comment.

Reply:
can we change this to a variadic argument that accepts one or more volume types to include?